pages/ksphere/konvoy/1.4/tutorials/addon-repositories/index.md

```diff
@@ -55,11 +55,11 @@ The folders in the addon repository have the following roles:
 - `metadata/` - Contains the static metadata for each addon in `addons/`.
 - `deployments/` - Contains the default addons specific to the version of Kubernetes in use.
 
-Here is a link to a [sample repository][sample_repo] you can experiment with and use as a template for your own addon repository. It contains two addons `awsebscsiprovisioner2` and `cockroachdb`.
+Here is a link to a [sample repository](https://github.com/realmbgl/my-addon-repo) you can experiment with and use as a template for your own addon repository. It contains two addons `awsebscsiprovisioner2` and `cockroachdb`.
```

Comment:
We want to point to a personal github repo?

Reply:
definitely not, this should have been fixed in https://github.com/mesosphere/konvoy/commit/8d6ebba192dce5d6db4ddf880f56055c963423ea